inverse_covariance/tests/clean_test.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from scipy import sparse
+
+from sklearn.utils.testing import assert_raises
+
+from inverse_covariance.clean import (
+   twoway_standardize
+)
+
+
+def custom_init(n_rows, n_cols, with_mean=False):
+    prng = np.random.RandomState(1)
+    X = prng.normal(0, np.ones(shape=(n_rows, n_cols)))
+    if with_mean:
+        mu = np.ones(shape=(n_rows, 1)) * \
+                prng.randint(1, 5, size=(1, n_cols))
+    else:
+        mu = np.zeros(shape=(n_rows, n_cols))
+    var_rows = prng.lognormal(2, 1, size=(n_rows, 1))
+    sqcov_rows = np.diag(np.sqrt(var_rows))
+    var_cols = prng.lognormal(2, 1, size=(1, n_cols))
+    sqcov_cols = np.diag(np.sqrt(var_cols))
+    return mu + sqcov_rows * X * sqcov_cols
+
+
```

Comment:
Can we add some tests for two_way_standardize that test that it does what's expected, not just the interface?  Like with a fixed input and expected output.